### PR TITLE
webserver: add implementation for uploading tnf config file and add all the fields

### PR DESCRIPTION
**Pull request by aabughosh**

### CODE REVIEW

**Review by greyerof — APPROVED** [review]

File: cnf-certification-test/webserver/index.html [hunk 1]
```diff
@@ -9,6 +10,9 @@
           sizes="any"
           href="https://ux.redhat.com/assets/logo-red-hat.svg">
     <link rel="stylesheet" href="https://ux.redhat.com/assets/packages/@rhds/elements/elements/rh-footer/rh-footer-lightdom.css">
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/2.1.1/jquery.min.js"></script>
```
Comment (sebrandon1) [inline cnf-certification-test/webserver/index.html:13]: JQuery [v3.7.1](https://github.com/jquery/jquery/releases/tag/3.7.1) is the latest release.  Should we be using that?

File: cnf-certification-test/webserver/index.html [hunk 2]
```diff
@@ -9,6 +10,9 @@
...
+   <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.0.13/css/all.css" integrity="sha384-DNOHZ68U8hZfKXOrtjWvjxusGo9WQnrNx2sqG0tfsghAvtVlRW3tvkXWZh58N9jp" crossorigin="anonymous">
```
Comment (sebrandon1) [inline cnf-certification-test/webserver/index.html:14]: Font-Awesome's latest release is [v6.4.2](https://github.com/FortAwesome/Font-Awesome/releases/tag/6.4.2).

**Review by sebrandon1 — COMMENTED** [review]
Just some comments about versions.

**Review by sebrandon1 — APPROVED** [review]
We can approve/merge this and address the updates in a secondary PR if needed.

**Pull request merged**